.eslintrc.json

```diff
@@ -245,5 +247,11 @@
         "sort-imports": 0,
         "template-curly-spacing": [1, "never"],
         "yield-star-spacing": [1, "before"]
-    }
+    },
+    "overrides": [
+        {
+          "files": ["*.ts", "*.tsx", ".*.ts", ".*.tsx"],
```

Comment:
Looks like some duplicates here? Also, we can probably remove tsx, since we are highly unlikely to add any JSX to this codebase.

Reply:
ok ill remove it